### PR TITLE
monitoring/ceph-mixin: fix typos

### PR DESCRIPTION
**Pull request by lonniebe**
fixing minor typos.

Signed-off-by: Lonnie Berry <lonchbe@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by tchaikov — APPROVED** [review]

File: monitoring/ceph-mixin/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All the Grafana dashboards are already generated in the `dashboards_out`
 directory and alerts in the `prometheus_alerts.yml` file.
 
 You can use the Grafana dashboards and alerts with Jsonnet like any other
-prometheus mixin. You can find more ressources about mixins in general on
+prometheus mixin. You can find more resources about mixins in general on
```
Comment (anthonyeleven) [inline monitoring/ceph-mixin/README.md:8]: Prometheus

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

**Review by anthonyeleven — APPROVED** [review]

File: monitoring/ceph-mixin/README.md (hunk 1 above)
Comment (lonniebe) [reply monitoring/ceph-mixin/README.md]: fixed it, thank you.

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Jenkins retest this please

**Comment by lonniebe** [comment]
@anthonyeleven I had some extra commits pushed accidently, fixed now.
Thank you for the quick review! 👍

**Comment by lonniebe** [comment]
jenkins test dashboard cephadm

**Comment by anthonyeleven** [comment]
I suspect that the Jenkins / check rig is broken, hence the errors.  I don’t think they’re related to your changes but we can’t merge until we resolve them

**Comment by anthonyeleven** [comment]
jenkins please retest this

**Comment by anthonyeleven** [comment]
jenkins retest this please

**Review by avanthakkar — APPROVED** [review]
LGTM!

**Comment by avanthakkar** [comment]
jenkins test dashboard

**Comment by avanthakkar** [comment]
@lonniebe  I think you need to fix unit tests 
`Unit Testing:  /home/jenkins-build/build/workspace/ceph-pull-requests/monitoring/ceph-mixin/tests_alerts/test_alerts.yml`
https://jenkins.ceph.com/job/ceph-pull-requests/99615/testReport/junit/projectroot.monitoring/ceph-mixin/run_promtool_unittests/

**Comment by avanthakkar** [comment]
jenkins test windows

**Comment by lonniebe** [comment]
jenkins retest this please

**Comment by nizamial09** [comment]
@lonniebe can you rebase this once more and then it'll be good to go!

**Comment by nizamial09** [comment]
jenkins test make check

**Pull request closed without merging**